functests/utils/profiles/profiles.go

```diff
@@ -5,8 +5,11 @@ import (
 	"fmt"
 	"reflect"
 
+	. "github.com/onsi/gomega"
+
 	testclient "github.com/openshift-kni/performance-addon-operators/functests/utils/client"
 	performancev1alpha1 "github.com/openshift-kni/performance-addon-operators/pkg/apis/performance/v1alpha1"
+	v1 "github.com/openshift/custom-resource-status/conditions/v1"
```

Comment:
Ah, I forgot about this package :+1: 
There are some useful funcs in it as well (e.g. `IsStatusConditionPresentAndEqual(), IsStatusConditionTrue()`...), maybe worth to refactor our "find condition" funcs to use them in a follow-up PR?

Reply:
Agree